### DISCUSSION TITLE
Upgrading from PostgreSQL 13 to PostgreSQL16

### DISCUSSION THREAD

**Discussion by CarrJuan** [opendcs/.github#5] [General]
Good afternoon.  The Bureau of Reclamation is upgrading from PostgreSQL 13 to PostgreSQL16, and we are trying to find out if we will have any issues with the DB in regard to OpenDCS 6.5 RC01.  We know we need to upgrade to OpenDCS 7.X.  Would just like to upgrad the DB first.  Anyone have any experience with PostgreSQL versions?

**Comment by MikeNeilson** [comment]
You *might* have to drop in a newer postgres-jdbc jar into the deps folder, but otherwise you shouldn't have any issues.

I've tested 7.x with postgres 16 and the schema is the same, at least for the purpose of "postgres". There's going to be some different columns, but no special features that were introduced between 13 and 16 that would be a problem.


Though as always, *TAKE A BACKUP FIRST*.
